raid.adoc

```diff
@@ -674,9 +682,20 @@ possible and must be used with extra care.
 
 If the disk is turned on (when prompted) during the script, it will
 automatically be "`Allowed`" by both this script and
-_refresh_secondary_, which also supports this feature.  This allows
-you to then run _refresh_secondary_ immediately without having to
-shutdown, turn the disk off, reboot, start the script, and turn the disk on.
+_refresh_secondary_, which also supports this feature. This allows you
+to then run _refresh_secondary_ immediately without having to
+shutdown, turn the disk off, reboot, start the script, and turn the
+disk on.
+
+The `-A` will also allow blanking of a disk that is too small to
+support the current RAID. This might be used to initialize a disk that
+will not be used in the current RAID. As before, use the `-A` option
+only will expert advice.
+
+The `-Z` option (for expert use only) will "`zap`" the partition
+tables with 1 MiB of zeros. Each additional `-Z` specified will double
```

Comment:
Technically it zaps the partition table and the start of the individual partitions as well.  The -Z doubling is only applicable to the partitions since the partition table is always less than 1MiB in size.